lib/cloudinary.rb

```diff
@@ -140,18 +128,34 @@ def self.app_root
   end
 
   private
-  
+
+  def self.config_from_env
+    # Heroku support
+    if ENV["CLOUDINARY_CLOUD_NAME"]
+      config_keys = ENV.keys.select! { |key| key.start_with? "CLOUDINARY_" }
+      config_keys -= ["CLOUDINARY_URL"] # handled separately
```

Comment:
Is this really "handled separately" or we just ignore it when explicit options are passed?